src/main/java/com/khutircraftubackend/config/SecurityConfig.java

```diff
@@ -46,8 +46,7 @@ public SecurityFilterChain filterChain(HttpSecurity http, UserDetailsServiceImpl
                         // Дозволяємо доступ до інших ресурсів
                         .requestMatchers("/v1/user/login", "/v1/user/register", "/v1/user/confirm", "/error").permitAll()
                         // Визначаємо доступ для різних ролей
-                        .requestMatchers("/products/**", "/sellers/**").permitAll()// Доступ для всіх
-                        .requestMatchers("/seller/**").hasRole("SELLER")
+                        .requestMatchers("/products/**").permitAll()// Доступ для всіх
```

Comment:
will `/sellers/**` and `/seller/**` be publicly accessible?
as per design we've got a section on the Home Page with "Our Sellers"
and potential buyer might need to see more details for a specific seller of a product without being authorized in the system

Reply:
Unfortunately, I don’t have access to the figma; I haven’t seen all the figs. Now sellers are available only to registered users, that is, to view information about themselves. When the endpoints are supplemented, then access can be configured. Now I see that access should be denied to everyone.

Reply:
ok, let's keep it this way 

i've just checked updated design for main (Home) page in figma again
it doesn't have that section with sellers